config/initializers/api.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+raise 'API_SECRET_KEY environment variable must be set' unless ENV['API_SECRET_KEY'].present? || Rails.env.test?
```

Comment:
don't think we need to raise it here, better to check on API call level, not every lcms needs API.

Reply:
Also we need to add it to `.env.template` + to https://github.com/learningtapestry/lcms-engine/blob/master/docs/env-variables.md

Reply:
Done